Source: Elias Pipping <pipping@exherbo.org>
Upstream: no
Reason: Uploading a file during src_test() is unacceptable
---
 Tests/CMakeLists.txt | 22 ++++++++++++----------
 1 file changed, 12 insertions(+), 10 deletions(-)

diff --git a/Tests/CMakeLists.txt b/Tests/CMakeLists.txt
index b404333..6a2c85b 100644
--- a/Tests/CMakeLists.txt
+++ b/Tests/CMakeLists.txt
@@ -1705,16 +1705,18 @@ ${CMake_BINARY_DIR}/bin/cmake -DVERSION=master -P ${CMake_SOURCE_DIR}/Utilities/
     PASS_REGULAR_EXPRESSION "Could not find executable"
     FAIL_REGULAR_EXPRESSION "SegFault")
 
-  configure_file(
-    "${CMake_SOURCE_DIR}/Tests/CTestTestUpload/test.cmake.in"
-    "${CMake_BINARY_DIR}/Tests/CTestTestUpload/test.cmake"
-    @ONLY ESCAPE_QUOTES)
-  add_test(CTestTestUpload ${CMAKE_CTEST_COMMAND}
-    -S "${CMake_BINARY_DIR}/Tests/CTestTestUpload/test.cmake" -V
-    --output-log "${CMake_BINARY_DIR}/Tests/CTestTestUpload/testOut.log"
-    )
-  set_tests_properties(CTestTestUpload PROPERTIES
-    PASS_REGULAR_EXPRESSION "Upload\\.xml")
+  # Exherbo: Test disabled because of network sandboxing
+  ######################################################
+  # configure_file(
+  #   "${CMake_SOURCE_DIR}/Tests/CTestTestUpload/test.cmake.in"
+  #   "${CMake_BINARY_DIR}/Tests/CTestTestUpload/test.cmake"
+  #   @ONLY ESCAPE_QUOTES)
+  # add_test(CTestTestUpload ${CMAKE_CTEST_COMMAND}
+  #   -S "${CMake_BINARY_DIR}/Tests/CTestTestUpload/test.cmake" -V
+  #   --output-log "${CMake_BINARY_DIR}/Tests/CTestTestUpload/testOut.log"
+  #   )
+  # set_tests_properties(CTestTestUpload PROPERTIES
+  #   PASS_REGULAR_EXPRESSION "Upload\\.xml")
 
   configure_file(
     "${CMake_SOURCE_DIR}/Tests/CTestTestConfigFileInBuildDir/test1.cmake.in"
-- 
1.8.1.2

